wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -895,13 +895,27 @@ private WorkspaceImpl doCreateWorkspace(WorkspaceConfig config,
     }
 
     private WorkspaceImpl getByKey(String key) throws NotFoundException, ServerException {
+
+        int lastColonIndex = key.indexOf(":");
```

Comment:
@sleshchenko https://github.com/eclipse/che/pull/4123/commits/07cd05da8823c6fe5f5d8f56f89bda98b939a107
is it ok ?